src/api/keystore.coffee

```diff
@@ -0,0 +1,171 @@
+Keystore = require('../model/keystore').Keystore
+Certificate = require('../model/keystore').Certificate
+Q = require 'q'
+logger = require 'winston'
+authorisation = require './authorisation'
+pem = require 'pem'
+
+logAndSetResponse = (ctx, status, msg, logLevel) ->
+  logger[logLevel] msg
+  ctx.body = msg
+  ctx.status = status
+
+exports.getServerCert = ->
+  # Must be admin
+  if authorisation.inGroup('admin', this.authenticated) is false
+    logAndSetResponse this, 'forbidden', "User #{this.authenticated.email} is not an admin, API access to getServerCert denied.", 'info'
+    return
+
+  try
+    keystoreDoc = yield Keystore.findOne().exec()
```

Comment:
Very minor point: but maybe it would be good to add a projection for `cert`? That way it won't read out the server key as well.

Probably not a big deal ito performance, but that way the key won't be sitting in memory or get transferred in the db call. Probably also not a big deal ito security, but you never know.